content/zh/docs/security/cve.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 常见漏洞与暴露（CVE）
+weight: 100
+default_lang_commit: f35b3300574b428f94dfeeca970d93c5a6ddbf35
+---
+
+以下是 [OpenTelemetry GitHub 组织](https://github.com/open-telemetry/)中所有代码仓库已报告的
+CVE 列表。原始数据存放在 [sig-security](https://github.com/open-telemetry/sig-security)
```

Comment:
FYI - the original (English) doc is misleading, sig-security needs to make changes here.
We don't have to block this PR though.

Reply:
@open-telemetry/sig-security-approvers FYI